### ISSUE TITLE
iobroker crashes during update of fullcalendar to version 2.x

### ISSUE THREAD

**Issue by Feuer-sturm** [ioBroker/ioBroker.fullcalendar#73]
**Describe the bug**  
iobroker crashes during update of fullcalendar to version 2.x

**To Reproduce**  
1. Install fullcalendar v1.2.0
2. Update to fullcalendar version 2.x e.g. 2.2.0 (but issue occurs as well with other 2.x versons as well)
3. at the end of the update the complete RAM and SWAP is used and iobroker crashes and restarts

**Expected behavior**  
update of adapter does not crash iobroker

**Screenshots & Logfiles**  

**Versions:**  
 - Adapter version: fullcalendar 2.2.0
 - JS-Controller version: 5.0.1-alpha.0-20230418-efa3e809
 - Node version: v18.16.0 and npm 9.5.1

**Screenshots & Logfiles**  
![Animation10](https://github.com/ioBroker/ioBroker.fullcalendar/assets/25422320/6e7417ee-dbf2-4065-898e-48b3ef6b005a)

**Additional context**  
- In this run iobroker did not crash but RAM and SWAP was completely used what is not the normal behavior during adapter updates
- In several other runs iobroker crashes and iobroker restarts
 ```
proxmox@ioBrokerTestsystem:~$ iob upgrade fullcalendar@2.2.0
Would you like to reinstall version 2.2.0 of fullcalendar now? [(y)es, (n)o]: y
Update fullcalendar from @2.2.0 to @2.2.0
NPM version: 9.5.1
 Installing iobroker.fullcalendar@2.2.0... (System call)

removed 105 packages in 15s

85 packages are looking for funding
  run `npm fund` for details
Adapter "system.adapter.vis.0" restarted.
Adapter "system.adapter.vis-2-beta.0" restarted.
upload [9] fullcalendar.admin /opt/iobroker/node_modules/iobroker.fullcalendar/admin/asset-manifest.json asset-manifest.json application/json
upload [8] fullcalendar.admin /opt/iobroker/node_modules/iobroker.fullcalendar/admin/favicon.ico favicon.ico image/vnd.microsoft.icon
upload [7] fullcalendar.admin /opt/iobroker/node_modules/iobroker.fullcalendar/admin/fullcalendar.png fullcalendar.png image/png
upload [6] fullcalendar.admin /opt/iobroker/node_modules/iobroker.fullcalendar/admin/manifest.json manifest.json application/json
upload [5] fullcalendar.admin /opt/iobroker/node_modules/iobroker.fullcalendar/admin/static/css/main.f9bd0fe2.css static/css/main.f9bd0fe2.css text/css
upload [4] fullcalendar.admin /opt/iobroker/node_modules/iobroker.fullcalendar/admin/static/css/main.f9bd0fe2.css.map static/css/main.f9bd0fe2.css.map applicati                                                                                           on/json
upload [3] fullcalendar.admin /opt/iobroker/node_modules/iobroker.fullcalendar/admin/static/js/main.1fa9299e.js static/js/main.1fa9299e.js application/javascrip                                                                                           t
upload [2] fullcalendar.admin /opt/iobroker/node_modules/iobroker.fullcalendar/admin/static/js/main.1fa9299e.js.LICENSE.txt static/js/main.1fa9299e.js.LICENSE.t                                                                                           xt text/plain
upload [1] fullcalendar.admin /opt/iobroker/node_modules/iobroker.fullcalendar/admin/static/js/main.1fa9299e.js.map static/js/main.1fa9299e.js.map application/j                                                                                           son
/usr/bin/iob: line 10: 25965 Killed sudo -H -u iobroker node /opt/iobroker/node_modules/iobroker.js-controller/iobroker.js "$@"
proxmox@ioBrokerTestsystem:~$
```
- If "iob stop" is executed before updating to "iobroker install fullcalendar" RAM and SWAP stays normal

**Comment by GermanBluefox** [comment]
Hm. I tried to downgrade fullcalendar to 1 and after creatio of some events (with running instance), I upgraded it to v2. Nothing unusual was spotted.

**Comment by Feuer-sturm** [comment]
@GermanBluefox I saw this behavior on one or two other adapters as well. First I thought it is a problem on my system with not to much RAM so I increased it to 4 GByte on my test system.

I just made the update to 2.2.6 and the system crashes again.
Bevor the update there was a RAM consumption about ~2 GByte
![grafik](https://github.com/ioBroker/ioBroker.fullcalendar/assets/25422320/68d91652-cd29-4a4a-843b-e54144a06e44)

Admin 6.6.1
js-controller 5.0.8

I justed double checked the behavior on my productive system with js-controller 4.0.xx and updated fullcalender from 2.2.2 to 2.2.6
I saw that the RAM consumption raised from 2,6 Gbyte (before starting the update) to 3,6 GByte but on this system the update was successfull.
I updated another adapter on my productive system as well (worx) and the RAM consumption was during the update 300 MByte higher than before.
